cdap-app-templates/cdap-etl/cdap-etl-api/pom.xml

```diff
@@ -35,6 +35,11 @@
       <groupId>co.cask.cdap</groupId>
       <artifactId>cdap-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
```

Comment:
Why avoid common-lang into this module? Is the idea to keep api as slim as possible?